components/atoms/Button/Button.js

```diff
@@ -0,0 +1,158 @@
+import Icon from '@/components/atoms/Icon'
+import cn from 'classnames'
+import NextLink from 'next/link'
+import PropTypes from 'prop-types'
+import React from 'react'
+import styles from './Button.module.css'
+
+/**
+ * Render the common inner part of the button component.
+ *
+ * @param {object}  props              The props object.
+ * @param {string}  props.icon         Optional icon.
+ * @param {boolean} props.iconOnly     Whether this button is an icon only.
+ * @param {string}  props.iconPosition Position to render the icon.
+ * @param {string}  props.text         Button text or aria-label.
+ * @return {Element}               The inside of the Button component.
```

Comment:
@itsamoreh Please fix the spacing/indentation here

Reply:
Fixed 85682ffdbb2c336eae2d8063bdb93ab10b040f37!